gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GaaSObservabilityEventV0.avsc

```diff
@@ -0,0 +1,162 @@
+{
+  "type": "record",
+  "name": "GaaSObservabilityEventV0",
+  "namespace" : "org.apache.gobblin.metrics",
+  "fields": [
+    {
+      "name": "timestamp",
+      "type": "long",
+      "doc": "Time at which event was created in millis"
+    }, {
+      "name" : "flowGroup",
+      "type" : "string",
+      "doc" : "Flow group for the GaaS flow",
+      "compliance" : "NONE"
+    }, {
+      "name" : "flowName",
+      "type" : "string",
+      "doc" : "Flow name for the GaaS flow",
+      "compliance" : "NONE"
+    }, {
+      "name" : "flowExecutionId",
+      "type" : "long",
+      "doc" : "Flow execution id for the GaaS flow",
+      "compliance" : "NONE"
+    }, {
+      "name": "jobSentToExecutor",
+      "type": "boolean",
+      "doc": "Whether or not this job was able to be sent to a job executor."
```

Comment:
this boolean is suspicious and perhaps could be derived from richer, more informative other fields.  do you see it as a sub-category of `FAILED` status?

maybe a `jobOrchestratedTime`?

Reply:
Yes I can repurpose this, what should the value be when the job is unable to be sent? -1?

Reply:
I wondered the same about the `jobEndTime`... that won't have a meaningful value in the absence of execution, will it?  in avro, I generally prefer a "non-complex" union w/ `null`, rather than a sentinel value like `-1`.